src/silx/resources/__init__.py

```diff
@@ -96,45 +97,36 @@
         _RESOURCES_DIR = _dir
 
 
-class _ResourceDirectory(object):
+class _ResourceDirectory(NamedTuple):
     """Store a source of resources"""
-
-    def __init__(self, package_name, package_path=None, forced_path=None):
-        if forced_path is None:
-            if package_path is None:
-                if pkg_resources is None:
-                    # In this case we have to compute the package path
-                    # Else it will not be used
-                    module = importlib.import_module(package_name)
-                    package_path = os.path.abspath(os.path.dirname(module.__file__))
-        self.package_name = package_name
-        self.package_path = package_path
-        self.forced_path = forced_path
+    package_name : str
+    forced_path : Optional[str] = None
```

Comment:
`package_path` is no longer used because there is no more a fallback as there was if `pkg_resources` was not available.